src/main/java/io/github/thebusybiscuit/slimefun4/core/services/holograms/HologramsService.java

```diff
@@ -114,6 +114,14 @@ private void purge() {
         }
     }
 
+    /**
+     * @see HologramsService#getHologram(Location, boolean)
+     */
+    @Nonnull
+    private Hologram getHologramOrCreate(@Nonnull Location loc) {
+        return getHologram(loc, true);
+    }
+
```

Comment:
```suggestion
```
I don't really see the point of making this method instead of just using `getHologram(loc, true)`. If it was used en masse _maybe_, but it's only used once and it's private so I don't think it's really necessary.

Reply:
Yeah, but the only difference in the `getHologram` and `getHologramOrCreate` methods is that the `getHologram` has `@Nullable` annotation and `getHologramOrCreate` has `@Nonnull` annotation. I just created this method for cleaner code and so that IntelliJ doesn't yell at me that i'm using a possibly null variable.